### ISSUE TITLE
[NETE2ESDK] Manually configuring the path does not work

### ISSUE THREAD

**Issue by v-xiaofchen** [dotnet/vscode-dotnet-runtime#1953]
### Describe the bug.

Manually configuring the path does not work

### Please attach log file(s) here if you have any.

The log file location should be copied to your clipboard if you came from VS Code.

### Reproduction Steps

1.Run the sample command ‘Install .NET SDK Globally via .NET Install Tool (Former Runtime Extension)‘.
2.Use the version it provided, which is 7.0.103. Hit enter.
3. Go to the extension settings by right clicking on the VSIX, and set the path to the return value on the terminal, like so: 
![image](https://github.com/user-attachments/assets/57092952-bc32-436f-87f1-d868c65ec3e5)
4.Click to edit the Existing Dotnet Path and add the following to the json file: 
```
 "dotnetAcquisitionExtension.existingDotnetPath": [ 
        { 
            "extensionId": "ms-dotnettools.sample-extension", 
            "path": "C:\\Program Files\\dotnet\\dotnet.exe" 
        } 
    ] 
```
Where path is the path of the installed .NET SDK. 
5.Install the .NET SDK as in step 1, but use a different version, such as 6.0.401
6.observe

### Expected Behavior

During this procedure, you should see the path it returns is the one you provided in the setting, and that no other install took place, so there should be no installer pop up or things like this. 
![image](https://github.com/user-attachments/assets/bad5b96d-570e-438f-be32-a28fc1d3800c)

### Exceptions & Errors (if any)

Manually configuring the path does not work and it continue to install the SDK.
![image](https://github.com/user-attachments/assets/d44cb15b-7c11-4ed7-b9ce-01ea37d7006b)

### .NET Install Tool Version

2.1.6

### Your operating system? Please provide more detail, such as the version and or distro above.

N/A

### VS Code Version

1.93.1

**Comment by v-xiaofchen** [comment]
@nagilson We did not find the new build or changed document either. Could you help to check it?

**Comment by nagilson** [comment]
The new build is in the same spot as last time, with the same name :) And the document you linked in the email is the one which was updated, 
![image](https://github.com/user-attachments/assets/7d77b7a7-7808-4c71-b23f-d68584bca710)

**Comment by v-xiaofchen** [comment]
I tried the updated 'Manually configure path' case in runtime test plan. Have some question need you check.
1. After editing the existing Dotnet path, the same runtime (e.g. "7.0") will download successfully, is it expected? The path setting can work when` allowInvalidPaths` is checked.
![1826133](https://github.com/user-attachments/assets/ffd90bae-53a9-4850-a328-6da5f76a635d)

3. You noted that the SDK install should now ignore the path setting. Did the test document in .NET Install Tool also need change?
![image](https://github.com/user-attachments/assets/d36c1820-f4dd-4657-bea0-aa30f43bc658)

**Comment by nagilson** [comment]
I don't think that is expected... let me look at the logs, thank you. The SDK one is a longer term decision. It should be updated.

**Comment by nagilson** [comment]
This is due to a bug and should be fixed in https://github.com/dotnet/vscode-dotnet-runtime/pull/1957
